### DISCUSSION TITLE
Can I translate Korean for personal use?

### DISCUSSION THREAD

**Discussion by Dankimhaejun** [graphql-rules/graphql-rules#119] [Q&A] [answered]
hi everyone!

First of all, Thanks for making this graphql-rules. It will be nice for beginners like myself.

If you don't mind, can I translate this document Korean for personal use?

I want to study this document and record my personal blog. (maybe this blog https://velog.io/@dankim ).

After you give me a permission, I will do next step.

Thanks for creating this nice document.

**Comment by RIP21** [comment]
@nodkz I guess, it's absolutely fine?
It will be even better if you'll enable @Dankimhaejun to contribute his translation to the website.

**Comment by nodkz** [comment, marked as answer]
@Dankimhaejun yeap sure it's fine to translate to Korean, with a backlink to this repo as the source. It will be greatly appreciated! ❤️

**Comment by RIP21** [comment]
@nodkz can we maybe put translation on the website itself (like React does let say)? So there are 2 versions available? Or it's not supported now?
